src/ORM/Filters/SearchFilter.php

```diff
@@ -474,7 +474,7 @@ protected function getCaseSensitive()
         } elseif (in_array('nocase', $modifiers ?? [])) {
             return false;
         } else {
-            $sensitive = self::config()->get('default_case_sensitive');
+            $sensitive = static::config()->get('default_case_sensitive');
```

Comment:
```suggestion
            $sensitive = SearchFilter::config()->get('default_case_sensitive');
```
This config was added in https://github.com/silverstripe/silverstripe-framework/pull/10925 and was very much intended to be a single value across the full class hierarchy.

Reply:
This goes along with the change you already made - https://github.com/silverstripe/silverstripe-framework/pull/11270#discussion_r1627033946

Reply:
Will change to SearchFilter::config()